### ISSUE TITLE
contribution guide for datasets

### ISSUE THREAD

**Issue by timkucera** [BorgwardtLab/proteinshake#119]
if someone wants to contribute, he/she could just issue a PR with the respective backend code. We could make the tests and release pipeline a bit more dynamic such that new datasets are automatically added to the pipeline once the PR is merged.

**Comment by timkucera** [comment]
duplicate of #29
